### PR TITLE
New ssh permissions strategy.

### PR DESCRIPTION
**Pull request by davidrkupton**
#### Fixes [insert bug/issue number]
Adhoc Development Changes

#### Changes proposed in this pull request:
* mounts local .ssh folder and then copies files to remote .ssh folder.

#### Add @mentions of the person or team responsible for reviewing proposed changes
@jimafisk @finneganh

### CODE REVIEW

File: scripts/init-docker-container.sh [hunk 1]
```diff
@@ -23,6 +23,9 @@ do
   ln -s "/host-repo/${p}" "./${p}"
 done
 
+# Uncomment next line to copy files in mounted ssh folder from host into container to ensure root:root permissions.
+# cp -rfu ~/host-ssh/ ~/.ssh/
```
Comment (finneganh) [inline scripts/init-docker-container.sh]: Wouldn’t this happen automatically by the `build:local` steps below?

Comment (davidrkupton) [reply scripts/init-docker-container.sh]: Yes.  It gets done in the local:sync just before each time we sync the database.  But if we don't want it to run each time we sync, then the best place to do it is here.

Comment (finneganh) [reply scripts/init-docker-container.sh]: Remember that if you’re running this script, the `build:local` commands below will completely reset your database, destroying anything that was synced in.

If you want to run this separately from `./task.sh` commands I’d recommend either noting it as a one-liner in `README.md` that you can run after bashing in, or add it to the `doit` script like some of the logging commands.

**Comment by finneganh** [comment]
Approved!

**Pull request merged**